blob-clients/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.bazaarvoice.emodb</groupId>
         <artifactId>emodb-parent</artifactId>
-        <version>6.4.12-SNAPSHOT</version>
+        <version>${project.version}</version>
```

Comment:
same as comment above

Reply:
will be removed here and added as part of above ticket